g3/generate/prompts/commit/creator.py

```diff
@@ -1,57 +1,88 @@
 import os
+import re
 
 import openai
 
 from g3.generate.prompts.commit.examples.node import node_sample
 from g3.generate.prompts.commit.examples.python import python_sample
 from g3.generate.prompts.commit.examples.ruby import ruby_sample
+from g3.generate.prompts.commit.system_messages.creator import (
+    Creator as SystemMessageCreator,
+)
+from g3.git.data import get_data_for_model
+from g3.main import config
+
+PY_PATTERN = re.compile(".*py")
+JS_PATTERN = re.compile(".*js")
+RB_PATTERN = re.compile(".*rb")
 
 
 class Creator:
-    def __init__(self, tone, jira, include):
-        self.tone = tone
-        self.jira = jira
-        self.include = include
+    def __init__(self):
         self.ruby_sample = ruby_sample
         self.node_sample = node_sample
         self.python_sample = python_sample
+        self.system_message_creator = SystemMessageCreator()  # TODO: add config for verbosity
```

Comment:
```suggestion
        self.system_message_creator = SystemMessageCreator()
```

I would prefer having a `commit_message_max_words` option that would be set once during `g configure` instead of `--verbose` flag that would result to different type of commit messages even from the same developer.

Defaulting to let's say up to `50` words for the whole commit message should be quick, right? If no let's default this config to even less words so that we achieve a latency for `g commit` up than 5-6 seconds.